patterns/testimonial-homepage.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Title: Homepage Testimonial
+ * Slug: course/testimonial-homepage
+ * Inserter: yes
+ * Categories: sensei-lms
+ */
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"80px","bottom":"80px"}}},"backgroundColor":"primary","textColor":"foreground","layout":{"type":"constrained","contentSize":"1000px"}} -->
+<div class="wp-block-group alignfull has-foreground-color has-primary-background-color has-text-color has-background" style="padding-top:80px;padding-bottom:80px"><!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}},"border":{"top":{"width":"0px","style":"none"},"right":{"width":"0px","style":"none"},"bottom":{"width":"0px","style":"none"},"left":{"color":"var:preset|color|tertiary","width":"1px"}}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group" style="border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-color:var(--wp--preset--color--tertiary);border-left-width:1px;padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"textColor":"tertiary"} -->
+<h3 class="has-tertiary-color has-text-color" style="text-transform:uppercase">What students say</h3>
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:columns {"style":{"spacing":{"blockGap":"20px"}},"textColor":"tertiary"} -->
+<div class="wp-block-columns has-tertiary-color has-text-color"><!-- wp:column {"verticalAlignment":"bottom","width":"41%"} -->
+<div class="wp-block-column is-vertically-aligned-bottom" style="flex-basis:41%"><!-- wp:image {"id":2150,"sizeSlug":"full","linkDestination":"none","style":{"border":{"radius":"8px","width":"1px"}}} -->
+<figure class="wp-block-image size-full has-custom-border"><img src="http://sentest.test/wp-content/themes/sensei-theme/assets/images/course-theme-testimonial.png" alt="" class="wp-image-2150" style="border-width:1px;border-radius:8px"/></figure>
```

Comment:
Replace `http://sentest.test` with `/`

Reply:
Updated as a part of [this](https://github.com/Automattic/sensei-theme/pull/44/commits/46592c3ecfd630ed62b365a18edd4411de136d34), thanks!